scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -918,6 +924,428 @@ cdef class TransformedDensityRejection(Method):
+
+    Polynomial interpolation based INVersion of CDF (PINV).
+
+    PINV is a variant of numerical inversion, where the inverse CDF is approximated
+    using Newton's interpolating formula. The interval ``[0,1]`` is split into several
+    subintervals. In each of these, the inverse CDF is constructed at nodes ``(CDF(x),x)``
+    for some points ``x`` in this subinterval. If the PDF is given, then the CDF is
+    computed numerically from the given PDF using adaptive Gauss-Lobatto integration with
+    5 points. Subintervals are split until the requested accuracy goal is reached.
+
+    The method is not exact, as it only produces random variates of the approximated
+    distribution. Nevertheless, the maximal tolerated approximation error can be set to
+    be the resolution (but, of course, is bounded by the machine precision). We use the
+    u-error ``|U - CDF(X)|`` to measure the error where ``X`` is the approximate
+    percentile corressponding to the quantile ``U`` i.e. ``X = approx_ppf(U)``. We call
+    the maximal tolerated u-error the u-resolution of the algorithm in the sequel.
+
+    Both the order of the interpolating polynomial and the u-resolution can be selected.
+    Note that very small values of the u-resolution are possible but increase the cost
+    for the setup step.
+
+    The interpolating polynomials have to be computed in a setup step. However, it only
+    works for distributions with bounded domain; for distributions with unbounded domain
+    the tails are cut off such that the probability for the tail regions is small compared
+    to the given u-resolution.
+
+    The construction of the interpolation polynomial only works when the PDF is unimodal
+    or when the PDF does not vanish between two modes.
+
+    Parameters
+    ----------
+    dist : object
+        An instance of a class with a ``pdf`` and optionally a ``cdf`` method.
+
+        * ``pdf``: PDF of the distribution. The signature of the PDF is expected to be:
+          ``def pdf(self, x: float) -> float``. i.e. the PDF should accept a Python
+          float and return a Python float. It doesn't need to integrate to 1 i.e. the
+          PDF doesn't need to be normalized.
+        * ``cdf``: CDF of the distribution. This method is optional. If provided, it
+          enables the calculation of "u-error". See `u_error`. Must have the same
+          signature as the PDF.
+
+    mode : float, optional
+        (Exact) Mode of the distribution. Default is ``None``.
+    center : float, optional
+        Approximate location of the mode or the mean of the distribution. This location
+        provides some information about the main part of the PDF and is used to avoid
+        numerical problems. Default is ``None``.
+    domain : list or tuple of length 2, optional
+        The support of the distribution. Must not contain `nan` values.
```

Comment:
I suggest to remove the comment on nans => we do not mention this for other parameters and it should be clear that nans do not make sense. you could mention that inf is admissible

Reply:
> I suggest to remove the comment on nans => we do not mention this for other parameters and it should be clear that nans do not make sense. you could mention that inf is admissible

You're right. Will make this change.

Reply:
Done.